CMakeModules/FindCLANG_FORMAT.cmake

```
@@ -44,11 +45,11 @@ if(CLANG_FORMAT_EXECUTABLE)
                   OUTPUT_VARIABLE clang_format_version
                   ERROR_QUIET OUTPUT_STRIP_TRAILING_WHITESPACE)
 
-  if(clang_format_version MATCHES "^clang-format version .*")
+  if(clang_format_version MATCHES "^[^ ]*[ ]*clang-format version .*")
```

```suggestion
  if(clang_format_version MATCHES "^.*clang-format version .*")
```

also `[ ]*` is the same as ` *`.

---

the whole regex thing is broken atm. The .* will match anything and thus the rest of the regex will fail

---

That is not right, with a middle pattern `.*` will only match the shortest pattern.